src/modules/coreHttpApi/openapi.yml

```diff
@@ -1039,9 +1039,8 @@ paths:
           schema:
             $ref: "#/components/schemas/AttributeID"
       responses:
-        200:
-          description: |
-            Success. Deleting an repository attribute.
+        204:
+          description: Success. Deleting an repository attribute.
```

Comment:
Deleting **a** RepositoryAttribute.

Reply:
yeah, I saw this too but was too lazy :D